backend/api/run.proto

```diff
@@ -188,6 +191,7 @@ message UnarchiveRunRequest {
 
 message DeleteRunRequest {
   string id = 1;
+  string namespace = 2;
```

Comment:
Looks like the run can be uniquely identified by the run ID, so the namespace might be unnecessary. Same with other API methods that work on existing runs.